examples/niscope_acquire_waveform/measurement.py

```diff
@@ -132,98 +134,77 @@ def cancel_callback():
             )
         )
 
-        if len(reservation.session_info) != 1:
-            measurement_service.context.abort(
-                grpc.StatusCode.INVALID_ARGUMENT,
-                f"Unsupported number of sessions: {len(reservation.session_info)}",
+        sessions = [
+            stack.enter_context(
+                _create_niscope_session(
+                    session_info, get_grpc_device_channel(measurement_service, niscope)
+                )
+            )
+            for session_info in reservation.session_info
+        ]
```

Comment:
This example should not support multiple sessions.
- NI-SCOPE supports channel expansion and automatically synchronizes instruments across the PXI trigger bus.
- With multiple independent sessions, it is impossible to map the trigger source from a DUT pin to a scope channel because each session has different scope channels.